### PR TITLE
Aavechan/gho borrow rate increase

### PR DESCRIPTION
**Pull request by Rozengarden**
Bump GHO borrow rate to 7.92% apr.

### CODE REVIEW

File: src/20240308_AaveV3Ethereum_GHOBorrowRateIncrease/GHOBorrowRateIncrease.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "GHOBorrowRateIncrease"
```
Comment (sakulstra) [inline src/20240308_AaveV3Ethereum_GHOBorrowRateIncrease/GHOBorrowRateIncrease.md:2]:
```suggestion
title: "GHO Borrow Rate Increase"
```

File: src/20240308_AaveV3Ethereum_GHOBorrowRateIncrease/GHOBorrowRateIncrease.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-increase-gho-borrow-rate-08-03-2024/16885"
+snapshot: "No snapshot for Direct-to-AIP"
```
Comment (sakulstra) [inline src/20240308_AaveV3Ethereum_GHOBorrowRateIncrease/GHOBorrowRateIncrease.md:5]:
```suggestion
```

File: src/20240308_AaveV3Ethereum_GHOBorrowRateIncrease/GHOBorrowRateIncrease.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+## Simple Summary
+
+This AIP will increase GHO borrow rate by 0.5% APR.
```
Comment (sakulstra) [inline src/20240308_AaveV3Ethereum_GHOBorrowRateIncrease/GHOBorrowRateIncrease.md:10]: should be by 0.7, no?

Comment (Rozengarden) [reply src/20240308_AaveV3Ethereum_GHOBorrowRateIncrease/GHOBorrowRateIncrease.md:9]: yes, should be .7%

Comment (Rozengarden) [reply src/20240308_AaveV3Ethereum_GHOBorrowRateIncrease/GHOBorrowRateIncrease.md]: changed the text to .7

**Review by rustboyar — APPROVED** [review]

**Pull request merged**